packages/toolkit/src/view/knowledge/components/CreateKnowledgeBaseCard.tsx

```diff
@@ -112,23 +113,23 @@ export const CreateKnowledgeBaseCard = ({
 
   const { accessToken, enabledQuery, selectedNamespace } = useInstillStore(useShallow(selector));
 
-  const { data: files } = useListKnowledgeBaseFiles({
+  const files = useListKnowledgeBaseFiles({
     namespaceId: selectedNamespace || null,
     knowledgeBaseId: knowledgeBase.kbId,
     accessToken: accessToken || null,
     enabled: enabledQuery && isHovered,
   });
 
   const chunkQueries = useQueries({
-    queries: (files || []).map((file) => ({
+    queries: (files.data || []).map((file) => ({
       queryKey: ["chunks", knowledgeBase.kbId, file.fileUid],
       queryFn: () => useGetAllChunks(
         accessToken || "",
         knowledgeBase.ownerName,
         knowledgeBase.kbId,
         file.fileUid
       ),
-      enabled: enabledQuery && isHovered && !!files && !!accessToken,
+      enabled: enabledQuery && isHovered && !!files.data && !!accessToken,
     })),
   });
```

Comment:
@thewbuk Please don't directly write query function in the component. Extract it as a standalone query function